js/src/common/Application.js

```diff
@@ -297,6 +306,11 @@ export default class Application {
       options.method = 'POST';
     }
 
+    // Save a reference to the xhr object, that can be used to later
+    // abort the request by calling this.activeRequests.abort(requestId)
+    const requestId = options.requestId || +new Date();
+    extend(options, 'config', (result, xhr) => this.activeRequests.set(requestId, xhr));
```

Comment:
Do we still need these changes?

Reply:
Yes, this part is needed to save a reference to the XHR object.